### PR TITLE
Anti mage buff

### PR DESCRIPTION
**Pull request by lovesmuffins**
Small buffs to late-game Anti-mage and overall buff to Anti-mage's ult which is terrible.

### CODE REVIEW

**Comment by TheUnholyWrath** [comment]
damage per mana already gets spell amped, you don't need to buff level 49. its the kind of shit that does 80k damage in an aoe if you carelessly buff it

**Comment by lovesmuffins** [comment]
> damage per mana already gets spell amped, you don't need to buff level 49. its the kind of shit that does 80k damage in an aoe if you carelessly buff it

Until I see heroes actually fall below 50% mana ever this ability deserves a buff. It's powerful but something that never happens.

File: game/scripts/npc/abilities/antimage_mana_void.txt [hunk 1]
```diff
@@ -47,7 +47,7 @@
       "01"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "mana_void_damage_per_mana"                       "0.8 0.95 1.1 1.55 2.3"
+        "mana_void_damage_per_mana"                       "1 1.3 1.6 2.0 3.0" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/antimage_mana_void.txt]: This doesn't need buffing if you buff mana break.

File: game/scripts/npc/abilities/antimage_mana_break.txt [hunk 2]
```diff
@@ -32,13 +32,13 @@
       }
       "03"
       {
-        "var_type"                                        "FIELD_INTEGER"
-        "mana_per_hit_pct"                                "1" //OAA
+        "var_type"                                        "FIELD_FLOAT" //OAA
+        "mana_per_hit_pct"                                "1 1 1 1 1.5 2.0" //OAA
```
[the remark this answers is not in the document]
Comment (DarkoniusXNG) [reply game/scripts/npc/abilities/antimage_mana_break.txt]: Revert to FIELD_INTEGER and put 1/1/2/2/3/4

**Pull request closed without merging**